packages/salesforcedx-vscode-soql/package.json

```diff
@@ -145,7 +147,7 @@
       ],
       "editor/title": [
         {
-          "when": "resourceLangId == soql",
+          "when": "resourceLangId == soql && resourceScheme == file",
```

Comment:
Toggling from the text editor to the soql builder with an unsaved file causes this issue shown below.  By checking the resourceScheme, we just don't show the toggle icon until the file is saved.  Which prevents users from toggling back and forth and getting into this situation.  Once saved, toggle works as expected.

https://user-images.githubusercontent.com/599418/112553326-e8c20f00-8d89-11eb-8ca8-ccd6abda10c5.mov